### PR TITLE
Add Fantom docs

### PR DESCRIPTION
**Pull request by akegaviar**

### CODE REVIEW

File: docs/platform/join-a-public-network.md [hunk 1]
```diff
@@ -70,6 +70,21 @@ The node status will change from **Pending** to **Running** once deployed.
 
 The node status will change from **Pending** to **Running** once deployed.
 
+## Join a Fantom network
+
+1. Select a [public chain project](/glossary/public-chain-project) and click **Get started** or **Join network**.
+1. Under **Blockchain protocol**, select **Fanto,**.
```
Comment (SozinM) [inline docs/platform/join-a-public-network.md:76]: typo

Comment (SozinM) [inline docs/platform/join-a-public-network.md:76]:
```suggestion
1. Under **Blockchain protocol**, select **Fantom**.
```

Comment (akegaviar) [reply docs/platform/join-a-public-network.md:76]: Thank you!

**Review by SozinM — APPROVED** [review]

File: docs/operations/fantom/tools.md [hunk 2]
```diff
@@ -0,0 +1,785 @@
...
+
+</template>
+</CodeSwitcher>
+
+3. Invoke any methods from [Web3 JavaScript API](https://web3js.readthedocs.io/). Make sure you replace the `eth` namespace with `ftm`.
+
+Example below demonstrates how to get the balance of an address in wei value and convert it to ether value:
+
+``` js
+> web3.fromWei(web3.ftm.getBalance("0x663f6ab861014056274128C69ff80caDCf048603"))
```
Comment (easeev) [inline docs/operations/fantom/tools.md:55]: Is `ftm` API namespace documented anywhere?

**Review by easeev — APPROVED** [review]
LGTM

File: docs/operations/fantom/tools.md (hunk 2 above)
Comment (akegaviar) [reply docs/operations/fantom/tools.md:55]: No (in their docs). It's just in some places in their docs and github I think. But I don't remember seeing explicit docs on this,

**Pull request merged**